dd-trace-ot/src/main/java/datadog/trace/common/writer/ZipkinV2Api.java

```diff
@@ -273,6 +276,14 @@ private static HttpURLConnection getHttpURLConnection(final String endpoint) thr
     return httpCon;
   }
 
+  private String truncatedString(String value) {
+    String encodedValue = value;
+    if (encodedValue.length() > recordedValueMaxLength) {
+      encodedValue = encodedValue.substring(0, recordedValueMaxLength) + "[...]";
```

Comment:
Is it ok that we append 5 characters so the value's length actually exceeds `recordedValueMaxLength`?

Reply:
I wondered about this too.  It could be poor ux to not signal having truncated a value as well as to not write as many recorded characters as specified by the configuration.  Since the max length is relatively arbitrary I thought this approach was preferable to the alternatives.

Reply:
Sounds reasonable.